src/test-utils/dom/prompt-input/index.ts

```diff
@@ -13,16 +13,27 @@ export default class PromptInputWrapper extends ComponentWrapper {
     return this.findByClassName<HTMLTextAreaElement>(testutilStyles.textarea)!;
   }
 
+  /**
+   * Finds the primary action button. Note that if you provide `primaryAction`
+   * then this will return null, you should use `findPrimaryActionSlot` instead.
+   */
   findActionButton(): ElementWrapper<HTMLButtonElement> {
     return this.findByClassName<HTMLButtonElement>(testutilStyles['action-button'])!;
   }
 
+  /**
+   * Finds the secondary actions slot. Note that, despite its typings, this may return null.
+   */
   findSecondaryActions(): ElementWrapper<HTMLDivElement> {
     return this.findByClassName<HTMLDivElement>(testutilStyles['secondary-actions'])!;
   }
 
-  findSecondaryContent(): ElementWrapper<HTMLDivElement> {
-    return this.findByClassName<HTMLDivElement>(testutilStyles['secondary-content'])!;
+  findSecondaryContent(): ElementWrapper<HTMLDivElement> | null {
```

Comment:
couldn't find any teams using this specific selector, and checked with dry run, so it should be safe to make this change